diffrax/_brownian/tree.py

```diff
@@ -161,67 +273,183 @@ def _cond_fun(_state):
             # jnp.abs(τ - state.s) > self.tol
             # Here, because we use quadratic splines to get better samples, we always
             # iterate down to the level of the spline.
```

Comment:
It is no longer a quadratic spline. I was considering replacing this whole comment with:

# While this evaluates to true, we will continue splitting the interval
# and thus moving down the tree.